app/networkapi/people/serializers.py

```diff
@@ -0,0 +1,48 @@
+from rest_framework import serializers
+
+from networkapi.people.models import (
+    Person,
+    Link,
+    Team,
+)
+
+class LinkSerializer(serializers.ModelSerializer):
+    """
+    Serializes a Link Model
+    """
+    class Meta:
+        model = Link
+        fields = (
+            'url',
+            'name',
+        )
+
+
+class TeamSerializer(serializers.ModelSerializer):
```

Comment:
no
delete the serializer completely. Just add a field to the `PersonSerializer` for team